Formula/clarinet.rb

```diff
@@ -1,9 +1,9 @@
 class Clarinet < Formula
   desc "Command-line tool and runtime for the Clarity smart contract language"
   homepage "https://github.com/hirosystems/clarinet"
-  url "https://github.com/hirosystems/clarinet/archive/v0.14.1.tar.gz"
-  sha256 "33ff6bc312f27fc450406091b67fac16be48b553b309f61538e8d0b16dbbb191"
-  license "GPL-3.0-only"
+  url "https://github.com/hirosystems/clarinet/archive/v0.14.2.tar.gz"
+  sha256 "a3335960bd7c2529b7a11744205030ce849ccaa539f34da57d13c1fff1379d38"
+  license "GPL-3.0"
```

Comment:
I don't think you should be changing this. While `Cargo.toml` does mention `GPL-3.0`, that is a deprecated identifier and one of `GPL-3.0-only` or `GPL-3.0-or-later` should be used. The repository doesn't seem to contain any references to "or later" in non-template text, so this should probably remain `GPL-3.0-only`.

Also, `brew audit --strict clarinet` should've complained about this change.

Reply:
Interesting, good to know. I ran the audit against the version change but forgot to rerun it after updating the license. I reverted the license change.